src/flux/app-config.coffee

```diff
@@ -4,11 +4,16 @@ flux = require 'flux-react'
 
 {makeSimpleStore} = require './helpers'
 
+MinimizedPattern = /tutor\.min-\d+/
```

Comment:
I'm not sure why you say: "and remove the learning-guide/ prefix in learning-guide-chart.coffee"

That needs to stay since those assets are in the learning-guide directory - or am I misunderstanding your intent?